Gems/AtomLyIntegration/EMotionFXAtom/Code/Source/AtomActorDebugDraw.h

```diff
@@ -87,6 +107,14 @@ namespace AZ::Render
             bool selected,              //!< Set to true if you want to render the axis using the selection color. */
             bool renderAxisName = false);
 
+
+
```

Comment:
uber nit: Are the extra line breaks required (I won't bother mentioning it again but I usually just have max 1 for blocks/separation, EMFX style may be different though).

Reply:
deleted